docs/TranslationGuide.md

```diff
@@ -1,49 +1,175 @@
+a few cases, when the value is long, the key is a KEY_STRING.
+
+When creating a new language file, make a copy of `en.json` with the new
+language code instead of `en`. The code must be in lower case. Then
+translate the file using English as the model.
+
+
+## gui-faq-LANG.md
+
+The FAQ documents holds questions and answers on Zonemaster, and there
+is one document per language, e.g. `gui-faq-en.md`.
+
+The files are located in the [docs/FAQ] folder, one file for each supported
+language.
+
+When creating a new FAQ file, make a copy of `gui-faq-en.md` with the new
+language code instead of `en`. The code must be in lower case. Then
+translate the file using English as the model.
+
+It is important to preserve the structure of the file. There must be a
+table of contents linking to the question plus answer below. The anchor
+names must be "q1" etc and nothing else. There are links in the code
+that assume this model.
 
-3. Ading the language link to the web interface
-The file that needs to be modified is locaded in the following folder:
 ```
-ZONEMASTER_DISTRIBUTION/zonemaster-gui/src/app/components/navigation.component.html
+1. [What is Zonemaster?](#q1)
+
+#### 1. What is Zonemaster? <a name="q1"></a>
 ```
 
+## Add the language to HTML code
+
+The language must be linked from the GUI. The new language must be added
+to [src/app/components/navigation/navigation.component.html]:
+
 In `navigation.component.html` :
 
- - Locate the div `<div class="lang">`
- - Add your language using the language tag: `<a lang='xx' (click)="setLanguage('xx')">XX</a> |`
-with xx, the language code in lower case and XX, the displayed language code.
- - Finally, for small (mobile) screen, add in the next select element:
- ``` 
+  * Locate the div `<div class="lang">`
+  * Add the language where `xx` is the language code in lower case, e.g.
+    `en` and `Yy` is the same language code with first letter in upper
+    case, e.g. `En`:
+
+```
+<a lang='xx' (click)="setLanguage('xx')">Yy</a> |
+```
+  * Add the language a second time where `xx` is the language code in
+    lower case, e.g. `sv`, and `Yyyyy` is the name of the same language
+    in that language, e.g. `Svenska`:
+
+```
  <option lang='xx' value='xx' [selected]="lang === 'xx'">
-  Xxxxx
+  Yyyyy
  </option>
  ```
- with Xxxxx, the full language name (En => English).
- 
- 
-4. Optionally, to change the default language update the following file: 
+
+Preserve the alphabetical order of the language codes.
+
+## Add the language in Type Script code
+
+The new language must be added to the following files:
+
+* [package.json]
+* [src/app/app.module.ts]
+* [src/app/components/navigation/navigation.component.ts]
+
+### package.json
+
+In `package.json` locate
+
 ```
-ZONEMASTER_DISTRIBUTION/zonemaster-gui/src/app/components/navigation.component.ts
+    "i18n:extract": "ngx-translate-extract --input ./src --output ./src/assets/i18n/{en,da,fr,sv}.json --key-as-default-value --clean --sort --format json"
```

Comment:
The `--output` are not listed in alphabetical order.

Reply:
Will fix.